requirements.txt

```diff
@@ -1,2 +1,3 @@
 Jinja2==2.8
+Django==1.11.4
```

Comment:
Yeah, you're right. It's not so good installing the whole framework just for using the templates. I was thinking about adding these extra renderers just in case someone that has already a Django application running would like to reuse the same templates. If you are going to do the changes remember to remove from the other file as well otherwise it will install Django when running pip install PyTerrier.